product-scenarios/2-Bridging-systems-that-communicate-in-different-protocols/2.1-HTTP-to-HTTP-protocol-translation/2.1.1-http-to-http-passthrough/src/test/java/org/wso2/carbon/esb/scenario/test/HttpMessageWithoutAlteringDownstreamTest.java

```diff
@@ -38,12 +38,17 @@
  * downstream without altering the content
  */
 public class HttpMessageWithoutAlteringDownstreamTest extends ScenarioTestBase{
-    private final String carFileName = "2_1_1_HttpMessageWithoutAlteringDownstreamTestCompositeApplication_1.0.0";
+   // private final String carFileName = "2_1_1_HttpMessageWithoutAlteringDownstreamTestCompositeApplication_1.0.0";
```

Comment:
Any reason to comment this? If this line is not necessary shall we remove the line without commenting?

Reply:
done